wandb/sdk/wandb_run.py

```diff
@@ -151,7 +151,7 @@ def __init__(
         self,
         interface: BackendSender,
         stop_polling_interval: int = 15,
-        retry_polling_interval: int = 1,
+        retry_polling_interval: int = 5,
```

Comment:
don't have strong opinions here but this will reduce user-side "liveness" of gql network errors from prior state (when we were just termlogging from the back, obviously we were not logging filestream errors before). also significantly changes the accuracy of the retry message, which says how many seconds from now we're gonna retry the request

Reply:
yeah, it is a bit weird that you get two messages, but seconds isnt what we care about i don't think.   With all the other things we have with queue management, i didnt want to add more stress to it.  lets go safe here, and we can make this better if we dont introduce new performance problems.

For instance, if the queue handling is blocked, polling at 1 second will stuff the queue with 5 times the messages that will have to be handled.   In the message polling design we have a solution for this, so we can ramp it back down to one once that goes in

Reply:
it doesn't generate a new error message on every poll, those are fed in independently on the retry loop but i'm fine with the 5s

Reply:
sorry, when i said "stuff the queue with 5 times the messages" i meant stuff the queue with network check PROTOCOL buffer messages.

What you really want here is:
```
send_message_to_internal_process_and_i_only_care_about_the_last_one(network_status_check)
```

But what we are doing is making a backlog of work, say we do this every 1 second for an hour and when the queue finally unblocks we process all 3600 of these proto messages even though only the last one will actually be handled.

Reply:
oh ok yeah that makes sense! we just have a lot of queues lol